src/services/message.service.ts

```diff
@@ -31,6 +31,46 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendMessage(
+    message: IMessage,
+    content: string,
+    reply: boolean,
+    argHeader?: string,
+    argFooter?: string,
+    argBreakChar?: string
+  ): Promise<Boolean> {
```

Comment:
im not really a fan of returning a boolean since we already have fulfillment/rejecting out of the box with promises. it makes more sense to make type `Promise<void>` and use reject/fufillment as flags for the status of the call